appscale/tools/remote_helper.py

```diff
@@ -254,7 +254,7 @@ def start_head_node(cls, options, my_id, node_layout):
       if agent.PARAM_REGION in params:
         additional_params[agent.PARAM_REGION] = params[agent.PARAM_REGION]
 
-    time.sleep(10)  # gives machines in cloud extra time to boot up
+      time.sleep(10)  # gives machines in cloud extra time to boot up
```

Comment:
Is this sleep actually necessary? And if so, do we know why 10 seconds is enough?